src/nrfx/hal/nrf_soc_if.h

```diff
@@ -95,8 +95,6 @@ extern void __SEV(void);
 extern void NVIC_SetPendingIRQ(IRQn_Type IRQn);
 extern void NVIC_ClearPendingIRQ(IRQn_Type IRQn);
 
-#define NRFX_ASSERT(...)
```

Comment:
Wouldn't it be better to add a #ifndef  both here and in the nrfx_glue? (so you don't need to comment out its uses)
Seems that these PRs anyway depend on each other, and it would be nicer to minimize the amount of differences around in this version of the HAL

Reply:
Yes, I think we can do that. For sure it will look better than those commented out assertions.

Reply:
Done.